src/dbus.c

```diff
@@ -1,15 +1,14 @@
 /* copyright 2013 Sascha Kruse and contributors (see LICENSE for licensing information) */
-
-#include <stdlib.h>
-#include <stdio.h>
+#include "dbus.h"
```

Comment:
That's a good point, and looking into this further I found the argument that having the header corresponding to the c file at the top also tests that each header can be included without any other includes.

So on second thought, leave it as is but leave an empty line between include layers i.e.
```
own header include
(empty line)
external includes
(empty line)
local includes
```